app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
```

Comment:
Code Style: You should use double quotes instead of single quotes for the string type hint.